packages/dds/tree/src/test/domains/json/json.ts

```diff
@@ -5,6 +5,18 @@
 
 import { makeRandom } from "@fluid-internal/stochastic-test-utils";
 
+interface Canada {
+    type: "FeatureCollection";
+    features: [{
+        type: "Feature";
+        properties: { name: "Canada"; };
+        geometry: {
+            type: "Polygon";
+            coordinates: [number, number][][];
+        };
+    }];
+}
+
```

Comment:
Following @seanimam's example of correctly typing the tree return from 'generateCanada()'.